bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -794,9 +799,22 @@ Full pathname to the inventory initialization control file.
 <entry id="use_fates_luh" type="logical" category="physics"
        group="clm_inparm" valid_values="" >
 If TRUE, enable use of land use harmonization (LUH) state and transition data from luh_timeseries file.
+This is enabled by default if fates_harvest_mode is set to use the raw LUH2 harvest data
 (Also, only valid for use_fates = true and is incompatible with transient runs currently.)
 </entry>
 
+<entry id="use_fates_lupft" type="logical" category="physics"
+       group="elm_inparm" valid_values="" value=".false.">
+If TRUE, enable use of fates land use x pft mapping data file.
+(Only valid for use_fates = true and is incompatible with transient runs currently.)
+</entry>
+
+<entry id="use_fates_potentialveg" type="logical" category="physics"
+       group="clm_inparm" valid_values="" >
+If TRUE, ignore the land-use state vector and transitions, and assert that all lands
```

Comment:
More explanation, and talk about how this is the spinup phase before transient is turned on. Using the term potential-veg is confusing here because it's what we use for paleo work. If this is part of the normal spinup and transition to transient -- should this be something that the user has to set?

Reply:
addressed via https://github.com/ESCOMP/CTSM/pull/2507/commits/32b2554189d6cbdb11361ed376eb53c4bb8d2857